### PR TITLE
Allow for scalar or size 1 `mu` in `MvNormal` and `MvStudentT`

### PR DESCRIPTION
**Pull request by ricardoV94**
Just for user friendliness, as this was accepted in V3

### CODE REVIEW

File: pymc/distributions/multivariate.py [hunk 1]
```diff
@@ -233,6 +233,9 @@ class MvNormal(Continuous):
     def dist(cls, mu, cov=None, tau=None, chol=None, lower=True, **kwargs):
         mu = at.as_tensor_variable(mu)
         cov = quaddist_matrix(cov, chol, tau, lower)
+        # Aesara is stricter about the shape of mu, than PyMC used to be
+        if mu.ndim < 2:
+            mu = at.broadcast_to(mu, cov.shape[0])
```
Comment (Sayam753) [inline pymc/distributions/multivariate.py]: AFAIK, `mu` will be broadcasted to shape (10, 3) in the case below in v3.

```python
>>> pm.MvNormal.dist(mu=np.ones((10, 1)), cov=np.eye(3))
```

More generally, `mu.shape` and `cov.shape[:-1]` should be broadcasted.
Is this case handled here as well?

Comment (ricardoV94) [reply pymc/distributions/multivariate.py]: Thanks, let me think about it

Comment (ricardoV94) [reply pymc/distributions/multivariate.py]: @Sayam753 I pushed an update which broadcasts `mu` to `cov[..., -1]`

**Comment by ricardoV94** [comment]
Failing test is flaky: https://github.com/pymc-devs/pymc/issues/5411

**Review by Sayam753 — APPROVED** [review]
LGTM 🚀 

We may need to revisit here after https://github.com/pymc-devs/pymc/issues/5383 gets solved, because if `cov` parameter supports batch dimensions, it may require broadcasting against `mu`.

**Comment by ricardoV94** [comment]
Thanks for reviewing @Sayam753 

Are you interested in picking some of #5383?

**Comment by Sayam753** [comment]
Yeah. Let's do it.

**Pull request merged**